src/components/molecules/UserList/UserList.tsx

```diff
@@ -32,7 +32,7 @@ export const UserList: React.FC<UserListProps> = ({
   limit = DEFAULT_USER_LIST_LIMIT,
   imageSize = 40,
   activity = "partying",
-  disableSeeAll = true,
+  disableSeeAll = false,
```

Comment:
Given you’re changing the default here it’s going to have impact in more places across the app than just the RoomModal. Have you found/asses the impact this will have on those locations? Where else is this used currently?

The minimal localised fix to the RoomModal would be to just pass through this prop on the RoomModa, rather than changing the default.

Reply:
I did click around and didn't spot oddity ... and couldn't see a use-case where I would want to see all potential 1000  (or even > 10) avatars.  If it is less than 11 (IIRC -- some magical number) it still shows all of them and without "See all", so I thought it would be actually of advantage generally.

I have tried to pass `disableSeeAll = {false}` in EventModal but for some reason it had no effect.

Reply:
In EventModal? Isn’t this in the RoomModal?

Reply:
yeap -- I thought it must have been that one since none other Modal has it
```
$> git grep UserList | grep Modal
src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx:import { UserList } from "components/molecules/UserList";
src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx:      <UserList
```
but looking there I actually see the `11` which I didn't saw back then -- must have been some wrong place ;)
```
      <UserList
        users={recentRoomUsers}
        limit={11}
        activity="in this room"
        attendanceBoost={room.attendanceBoost}
      />
```
so , reverted default (although still don't quite grasp why by default to display all possible 100s) and changed to have it only in RoomModal - seems to work. pushed

Reply:
Changing the default may eventually make sense, but without the capacity to assess the impact across all places, making a smaller localised change is easier.